### PR TITLE
Frmw1245 remove3.xdeprecations

### PR DESCRIPTION
**Pull request by cathysiller**
JIRA: https://jira.rax.io/browse/FRMW-1245

### LGTMs
- [x] Dev LGTM
- [x] Dev+Vidyo LGTM

### CODE REVIEW

File: src/elements/Tables/templates/rxPaginate.html [hunk 1]
```diff
@@ -13,7 +13,7 @@
           <a ng-click="pageTracking.goToPrevPage()" ng-hide="pageTracking.isFirstPage()">« Prev</a>
         </div>
 
-        <div ng-repeat="n in pageTracking | Page" ng-class="{active: pageTracking.isPage(n), 'page-number-last': pageTracking.isPageNTheLastPage(n)}" class="pagination-page">
+        <div ng-repeat="n in pageTracking | rxPager" ng-class="{active: pageTracking.isPage(n), 'page-number-last': pageTracking.isPageNTheLastPage(n)}" class="pagination-page">
```
Comment (CITguy) [inline src/elements/Tables/templates/rxPaginate.html]: Note: The `rxPager` filter will be refactored into `rxPageTracker` functionality when the `rxPaginate` directive functionality/display is updated.

File: src/utilities/rxApply/scripts/rxApply.js [hunk 2]
```diff
@@ -2,7 +2,6 @@
     angular
         .module('encore.ui.utilities')
         .filter('rxApply', rxApplyFilter)
```
Comment (CITguy) [inline src/utilities/rxApply/scripts/rxApply.js]: Please end line with semicolon.

File: src/utilities/rxPaginate/scripts/rxPaginate.js [hunk 3]
```diff
@@ -2,7 +2,6 @@
     angular
         .module('encore.ui.utilities')
         .filter('rxPaginate', rxPaginateFilter)
```
Comment (CITguy) [inline src/utilities/rxPaginate/scripts/rxPaginate.js]: Please end line with semicolon.

File: src/utilities/rxSelectFilter/scripts/rxSelectFilter.js [hunk 4]
```diff
@@ -65,7 +65,7 @@ angular.module('encore.ui.utilities')
  * Instances of `rxSelectFilter` have an `applyTo()` method, which applies the
  * filter's internal state of selected options to the array. This will not
  * often be called directly, but instead used by the
- * {@link utilities.filter:Apply Apply} filter. As stated previously,
+ * {@link utilities.filter:rxApply Apply} filter. As stated previously,
```
Comment (CITguy) [inline src/utilities/rxSelectFilter/scripts/rxSelectFilter.js]: Update link text to `rxApply`, too.

```
{@link utilities.filter:rxApply rxApply}
```

File: src/utilities/rxTitleize/scripts/rxTitleize.js [hunk 5]
```diff
@@ -2,7 +2,6 @@
     angular
         .module('encore.ui.utilities')
         .filter('rxTitleize', rxTitleizeFilter)
```
Comment (CITguy) [inline src/utilities/rxTitleize/scripts/rxTitleize.js]: Please end line with semicolon.

File: src/utilities/rxXor/scripts/rxXor.js [hunk 6]
```diff
@@ -2,7 +2,6 @@
     angular
         .module('encore.ui.utilities')
         .filter('rxXor', rxXorFilter)
```
Comment (CITguy) [inline src/utilities/rxXor/scripts/rxXor.js]: Please end line with semicolon.

File: src/elements/Tables/templates/rxPaginate.html (hunk 1 above)
Comment (cathysiller) [inline src/elements/Tables/templates/rxPaginate.html]: Noted

**Comment by cynprudence** [comment]
LGTM

**Pull request merged**